src/sentry/static/sentry/app/components/modals/integrationDetailsModal.jsx

```diff
@@ -0,0 +1,121 @@
+import {Box, Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import marked from 'marked';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import AddIntegrationButton from 'app/views/organizationIntegrations/addIntegrationButton';
+import Alert from 'app/components/alert';
+import Button from 'app/components/buttons/button';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import space from 'app/styles/space';
+
+class IntegrationDetailsModal extends React.Component {
+  static propTypes = {
+    closeModal: PropTypes.func.isRequired,
+    onAddIntegration: PropTypes.func.isRequired,
+    provider: PropTypes.object.isRequired,
+  };
+
+  onAddIntegration = integration => {
+    this.props.closeModal();
+    this.props.onAddIntegration(integration);
+  };
+
+  render() {
+    const {provider, closeModal} = this.props;
+    const {metadata} = provider;
+    const description = marked(metadata.description);
+
+    return (
+      <React.Fragment>
+        <Flex align="center" mb={2}>
+          <PluginIcon pluginId={provider.key} size={32} />
+          <ProviderName>{t('%s Integration', provider.name)}</ProviderName>
+        </Flex>
+        <Description dangerouslySetInnerHTML={{__html: description}} />
+        <Metadata>
+          <AuthorName flex={1}>{t('By %s', provider.metadata.author)}</AuthorName>
+          <Box>
+            <a href={metadata.source_url}>{t('View Source')}</a>
```

Comment:
Is `source_url` == plugin repo? And what is `issue_url`?

Would these be better as `ExternalLink`s?

Reply:
Yeah these would probably be better as `ExternalLink`s. These two things are similar to how plugins specify the author and issue links. Maybe something @saragilford could weigh in on, I pretty much just carried this over from the oldstyle plguins

> Is source_url == plugin repo

Not necessarily, since the integrations we're building right now are part of core sentry, they just link into the github tree.

Reply:
dbbb148

Reply:
Also I'll go back later and clean up the keys in the backend to get rid of the underscores.